interop/src/main.rs

```diff
@@ -176,6 +182,50 @@ fn run(log: Logger, options: Opt) -> Result<()> {
         }
     }
 
+    let mut h3 = false;
+    let result = runtime.block_on(
+        endpoint
+            .connect_with(&client_config, &remote, &options.host)?
+            .map_err(|e| format_err!("failed to connect: {}", e))
+            .and_then(|conn| {
+                assert!(state.lock().unwrap().saw_cert);
+                handshake = true;
```

Comment:
Copy-paste error? These two lines are redundant to their prior instance and should be removed here.

Reply:
fixed in 4ee65b9